api/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-slim as application
+FROM python:3.8 as application
```

Comment:
Why do we need this @matthewelwell ?

Reply:
I was having issues even with the postgres client but I think it might be arm related so you can take it out and see if it works.